File: hospital/AMarotoChivite-HospitalTest/AMarotoChivite-HospitalTest-Java/src/main/java/HospitalCollections/utils/randomData.java
package HospitalCollections.utils;

import java.util.Random;

public class randomData {

    public static String randomDni() {

        final String LETTERS = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
        final int LETTERS_LENGTH = LETTERS.length();

        Random random = new Random();
        StringBuilder sb = new StringBuilder();

        // Generamos 8 random dígitos
        for (int i = 0; i < 8; i++) {
            sb.append(random.nextInt(10));
        }

        sb.append(LETTERS.charAt(random.nextInt(LETTERS_LENGTH)));

        return sb.toString();
    }
}

File: hospital/AMarotoChivite-HospitalTest/AMarotoChivite-HospitalTest-Kotlin/src/main/kotlin/HospitalCollections/utils/randomData.kt
package HospitalCollections.utils

import java.util.*

object randomData {
    fun randomDni(): String {
        val LETTERS = "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
        val LETTERS_LENGTH = LETTERS.length
        val random = Random()
        val sb = StringBuilder()

        // Generamos 8 random dígitos
        for (i in 0..7) {
            sb.append(random.nextInt(10))
        }
        sb.append(LETTERS[random.nextInt(LETTERS_LENGTH)])
        return sb.toString()
    }
}